packages/@react-stately/collections/src/CollectionManager.ts

```diff
@@ -133,7 +133,7 @@ export class CollectionManager<T extends object, V, W> {
     this._transactionQueue = [];
 
     // Set options from passed object if given
-    this.transitionDuration = options.transitionDuration || 500;
+    this.transitionDuration = isNaN(options.transitionDuration) ? 500 : options.transitionDuration;
```

Comment:
you could do `options.transitionDuration ?? 500`

Reply:
WHAT IS THIS MAGIC!?

thanks